pkg/minikube/translate/translations/fr-FR.json

```diff
@@ -0,0 +1,90 @@
+	"For more information, see:": " ",
+	"If the above advice does not help, please let us know: ": " ",
+	"Ignoring --vm-driver=%s, as the existing %q VM was created using the %s driver.": " ",
+	"Kubernetes downgrade is not supported, will continue to use %v": " ",
+	"Launching Kubernetes ... ": " ",
+	"None of known repositories in your location is accessible. Use %s as fallback.": " ",
+	"None of known repositories is accessible. Consider specifying an alternative image repository with --image-repository flag": " ",
+	"Please enter a value:": " ",
+	"Powering off %q via SSH ...": " ",
+	"Pulling images ...": " ",
+	"Re-using the currently running %s VM for %q ...": " ",
+	"Related issues:": " ",
+	"Relaunching Kubernetes %s using %s ... ": " ",
+	"Requested disk size (%dMB) is less than minimum of %dMB": " ",
+	"Restarting existing %s VM for %q ...": " ",
+	"Sorry that minikube crashed. If this was unexpected, we would love to hear from you:": " ",
+	"Sorry, the --gpu feature is currently only supported with --vm-driver=kvm2": " ",
+	"Sorry, the --hidden feature is currently only supported with --vm-driver=kvm2": " ",
+	"Stopping %q in %s ...": " ",
+	"The 'none' driver provides limited isolation and may reduce system security and reliability.": " ",
+	"There is a newer version of minikube available (%s%s).  Download it here:\n%s%s\n\t\t\nTo disable this notification, run the following:\nminikube config set WantUpdateNotification false\n\t\t": " ",
+	"These changes will take effect upon a minikube delete and then a minikube start": " ",
+	"This can also be done automatically by setting the env var CHANGE_MINIKUBE_NONE_USER=true": " ",
+	"Tip: Use 'minikube start -p \u003cname\u003e' to create a new cluster, or 'minikube delete' to delete this one.": " ",
+	"To connect to this cluster, use: kubectl --context=%s": " ",
+	"To switch drivers, you may create a new VM using `minikube start -p \u003cname\u003e --vm-driver=%s`": " ",
+	"To use kubectl or minikube commands as your own user, you may": " ",
+	"Unable to bind flags": " ",
+	"Unable to get VM IP address": " ",
+	"Unable to load cached images from config file.": " ",
+	"Unable to load cached images: %v": " ",
+	"Unable to load config: %v": " ",
+	"Unable to parse %q: %v": " ",
+	"Unable to pull images, which may be OK: %v": " ",
+	"Unable to start VM": " ",
+	"Verifying:": " ",
+	"Wait failed": " ",
+	"Waiting for SSH access ...": "Attendant acces SSH ...",
+	"checking main repository and mirrors for images": " ",
+	"error getting driver": " ",
+	"kubectl and minikube configuration will be stored in %s": " ",
+	"minikube %s on %s (%s)": " ",
+	"minikube is not running, so the service cannot be accessed": " ",
+	"minikube will upgrade the local cluster from Kubernetes %s to %s": " ",
+	"need to relocate them. For example, to overwrite your own settings:": " ",
+	"sudo chown -R $USER $HOME/.kube $HOME/.minikube": " ",
```

Comment:
optional: We should probably just stop telling users to run commands, but I wonder if in the mean time, we should just filter out strings starting with "sudo"? feel free to ignore.